volume/volume.go

```diff
@@ -29,33 +39,78 @@ type Volume interface {
 	Unmount() error
 }
 
-// read-write modes
-var rwModes = map[string]bool{
-	"rw":   true,
-	"rw,Z": true,
-	"rw,z": true,
-	"z,rw": true,
-	"Z,rw": true,
-	"Z":    true,
-	"z":    true,
+// MountPoint is the intersection point between a volume and a container. It
+// specifies which volume is to be used and where inside a container it should
+// be mounted.
+type MountPoint struct {
```

Comment:
It is the lowest common denominator in terms of the dependency tree between packages. I tried it in several places, this was the logical end location.